Parse/PFFile.m

```diff
@@ -469,22 +482,6 @@ - (PFFileState *)_fileState {
     return state;
 }
 
-- (NSString *)stagedFilePath {
```

Comment:
You also need to remove the property on top and convert the usages of this property, as this will always be `nil` at this point.